docs/api/covidcast_inactive_signals.md

```diff
@@ -7,7 +7,4 @@ nav_order: 7
 
 # COVIDcast Inactive Signals
 
-These signals are currently not updated, because the sources have become
-unavailable, they have been replaced by other sources, or because additional
-work is required for us to continue to update them. Some of these sources may
-return in the coming months.
+These signals are currently not updated because the sources have become unavailable, other sources have replaced them, other sources have replaced them, or additional work is required for us to continue updating them. Some of these sources may return in the coming months.
```

Comment:
```suggestion
These signals are currently not updated because the sources have become unavailable, other sources have replaced them, or additional work is required for us to continue updating them. Some of these sources may return in the coming months.
```